csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -3616,7 +3634,16 @@ Secondly, the program for all items of:
 
 * type `/definitions/version_t`: If any element doesn't match the semantic versioning, replace the all elements of type `/definitions/version_t` with the corresponding integer version. For that, CVRF CSAF converter sorts the items of `/document/tracking/revision_history` by `number` ascending according to the rules of CVRF. Then, it replaces the value of `number` with the index number in the array (starting with 1). The value of `/document/tracking/version` is replaced by value of `number` of the corresponding revision item. The match must be calculated by the original values used in the CVRF document.
 * `/document/acknowledgments[]/organization` and `/vulnerabilities[]/acknowledgments[]/organization`: If more than one cvrf:Organization instance is given, the CVRF CSAF converter converts the first one into the `organization`. In addition the converter outputs a warning that information might be lost during conversion of document or vulnerability acknowledgment.
-* `/document/publisher/name`: Sets the value as given in the configuration of the program or the corresponding argument the program was invoked with. If both values are present, the program should prefer the latter one.
+* `/document/publisher/name` and `/document/publisher/namespace`: Sets the value as given in
+
+  1. the corresponding argument from the command line,
+  2. the config file specified on the command line,
+  3. the corresponding environment variable,
+  4. the local configuration of the program,
+  5. the user configuration of the program or
+  6. the system configuration of the program.
+
+  If more than one of these value is present, the program should prefer the one with the lowest number in the list above.
```

Comment:
I do not understand why we explain possible configuration options in so much detail here. Depending on different use cases and requirements, this list might look entirely different, so this section might generate more questions than it will help.
I would suggest to shorten as follows:
```
* `/document/publisher/name` and `/document/publisher/namespace`: Both values are not contained in the right format in the original CVRF document. The converter MUST provide sensible values itself (e.g. by requesting user input).
```

Reply:
It really helps interoperability a lot since you then can change from one CVRF-CSAF converter to a different one fairly quickly.

BTW: `The converter MUST provide sensible values itself (e.g. by requesting user input).` This includes hard coded values which should not be compliant.

Reply:
> It really helps interoperability a lot since you then can change from one CVRF-CSAF converter to a different one fairly quickly.

I would very much hope to be able to select one from multiple available converters. However, I doubt that we will have any positive effect by going into such implementation details.

Reply:
I change the wording and remove the configuration detail.